SuperBuild/External_JsonCpp.cmake

```diff
@@ -0,0 +1,73 @@
+
+set( proj JsonCpp )
+
+# Set dependency list
+set(${proj}_DEPENDENCIES "")
+
+# Include dependent projects if any
+ExternalProject_Include_Dependencies(${proj} PROJECT_VAR proj DEPENDS_VAR ${proj}_DEPENDENCIES)
+
+# Sanity checks
+if(${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
+  message(FATAL_ERROR "Enabling ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj} is not supported !")
+endif()
+if(DEFINED ${proj}_DIR AND NOT EXISTS ${${proj}_DIR})
+  message(FATAL_ERROR "${proj}_DIR variable is defined but corresponds to nonexistent directory")
+endif()
+
+if(NOT DEFINED ${proj}_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
+
+  set(EXTERNAL_PROJECT_OPTIONAL_ARGS)
+
+  if(NOT DEFINED git_protocol)
+    set(git_protocol "git")
+  endif()
+
+  if (NOT WIN32)
+    list(APPEND EXTERNAL_PROJECT_OPTIONAL_ARGS
+      -DCMAKE_BUILD_TYPE:STRING=${CMAKE_BUILD_TYPE}
+      )
+  endif()
+
+  ExternalProject_Add(${proj}
+    ${${proj}_EP_ARGS}
+    GIT_REPOSITORY "${git_protocol}://github.com/KitwareMedical/jsoncpp.git"
+    GIT_TAG "110d054227e9eb63faad48a1fb6a828ad0670e61"
+    SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj}
+    BINARY_DIR ${proj}-build
+    CMAKE_CACHE_ARGS
+      -DCMAKE_CXX_COMPILER:FILEPATH=${CMAKE_CXX_COMPILER}
+      -DCMAKE_CXX_FLAGS:STRING=${ep_common_cxx_flags}
+      -DCMAKE_C_COMPILER:FILEPATH=${CMAKE_C_COMPILER}
+      -DCMAKE_C_FLAGS:STRING=${ep_common_c_flags} # Unused
+      -DBUILD_TESTING:BOOL=OFF
+      -DJSONCPP_WITH_TESTS:BOOL=OFF
+      -DJSONCPP_WITH_POST_BUILD_UNITTEST:BOOL=OFF
+      -DJSONCPP_WITH_WARNING_AS_ERROR:BOOL=OFF
+      -DBUILD_SHARED_LIBS:BOOL=ON
```

Comment:
Since it is build as shared, make also sure to update SlicerCPackBundleFixup.cmake like it is done here: https://github.com/Slicer/Slicer/blob/26f50e4b5c5b007419bc1e8c8299fce09ae617a9/CMake/SlicerCPackBundleFixup.cmake.in#L102-L107